pex/interpreter.py

```diff
@@ -261,8 +261,10 @@ def __str__(self):
     )
 
   def __repr__(self):
-    return 'PythonIdentity(%r, %s, %s, %s)' % (
-      self._interpreter,
+    return 'PythonIdentity(%r, %r, %r, %r, %r, %r)' % (
```

Comment:
Oh how I wish we could use f-strings. 

I would say we should change this to new style string interpolation, but I know PEX always uses old style so this is fine.

Reply:
It does not - there is a mix. As a rule, pex or any other project, I try to stick to the conventions in the file unless on a pure cleaning change.